app/screens/transfer-signing-screen/transfer-signing-screen.tsx

```diff
@@ -38,6 +41,7 @@ export class TransferSigningScreen extends React.Component<TransferSigningScreen
   }
 
   private onPressConfirmButton = () => {
+    logAnalyticsEvent('TransferConfirmTx')
```

Comment:
Should split 2 events here

IF success `TransferConfirmTx` ELSE `TransferSign`